Sources/JSONable.swift

```diff
@@ -222,7 +71,10 @@ extension Int: JSONable {
     public static func fromJSON(_ x: JSONValue) -> Int? {
         switch x {
         case let .number(n):
-            return Int(n)
+            switch n {
+            case .int(let i): return Int(exactly: i)
+            case .fraction(let f): return Int(exactly: f)
```

Comment:
Ditto

Reply:
well, here it would just be plain wrong. `Int.fromJSON` should only work if the value can actually parse as an int, which a fractional value would not.

Reply:
You're right, I didn't see we were in an `Int` extension here.